capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -0,0 +1,108 @@
+package com.google.univiz.scorecard;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * ScorecardData is a class representing a single university/college/institution from the parsed
+ * JSON data directly from the CollegeScorecard API.
+ */
+@AutoValue
+public abstract class ScorecardData {
+
+  /**
+   * Creates a new ScorecardData
```

Comment:
It would be worth referencing where these param descriptions are coming from. That way if someone needs to add a field in the future it will be easy to find...